recipes/mimalloc/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+from conans import ConanFile, CMake, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+
+class MimallocConan(ConanFile):
+    name = "mimalloc"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/microsoft/mimalloc"
+    description = "mimalloc is a compact general purpose allocator with excellent performance."
+    topics = ("libraries", "c", "c++", "malloc", "allocator")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False],
+               "shared": [True, False],
+               "override": [True, False]}
+    default_options = {"fPIC": True,
+                       "shared": False,
+                       "override": True}
+    generators = "cmake", "visual_studio"
+    exports_sources = "patches/**"
+    _cmake_i = None
+    _msbuild_i = None
+
+    @property
+    def _cmake_install_folder(self):
+        p1 = self.version.find(".")
+        if p1 != -1:
+            p2 = self.version.find(".", p1 + 1)
+            if p2 != -1:
+                return self.name + "-" + self.version[0:p2]
+            else:
+                return self.name + "-" + self.version
+        else:
+            return self.name + "-" + self.version
```

Comment:
https://github.com/microsoft/mimalloc/blob/a9686d6ecf00e4467e772f7c0b4ef76a15f325f6/cmake/mimalloc-config-version.cmake#L1-L3
I think version is just `major + "." + minor`

```suggestion
        version = tools.Version(self.version)
        return "{}.{}".format(version.major, version.minor)
```

Correct me if I'm wrong please.